### PR TITLE
Sorting of PDS members by date created

### PR DESCRIPTION
**Pull request by SanthoshiBoyina**
## Proposed changes

To add an additional PDS member sorting option that allows users to sort by Date Created.

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [x] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [x] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Comment by zFernand0** [comment]
Do we need a 2.16.0 milestone?
This seems to be going into the `next` branch.

Also, this seems like a port of:
- https://github.com/zowe/vscode-extension-for-zowe/pull/2707 

🥳

**Review by zFernand0 — DISMISSED** [review]
Left some minor suggestion. 😅 
And thanks for porting 😋 

LGTM! 😋

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @SanthoshiBoyina! Please address Fernando's comment about changelog 🙂

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
+- Implemented sorting of PDS members by date created [#2707](https://github.com/zowe/vscode-extension-for-zowe/pull/2707)
```
Comment (traeok) [inline packages/zowe-explorer/CHANGELOG.md:9]: Is there an equivalent issue we can link to instead of the pull request for v2?

**Review by traeok — APPROVED** [review]
LGTM, thanks @SanthoshiBoyina!
Looks like there are some merge conflicts - will re-approve (if needed) after those are resolved.

File: packages/zowe-explorer/CHANGELOG.md (hunk 1 above)
Comment (adam-wolfe) [reply packages/zowe-explorer/CHANGELOG.md:9]: https://github.com/zowe/vscode-extension-for-zowe/issues/2565

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 

Thanks @SanthoshiBoyina 🥳

**Review by adam-wolfe — APPROVED** [review]
LGTM

**Comment by JillieBeanSim** [comment]
Do we have an issue for the circular dependencies that came into this from the next branch?

**Comment by traeok** [comment]
> Do we have an issue for the circular dependencies that came into this from the next branch?

@t1m0thyj is working on resolving these in https://github.com/zowe/vscode-extension-for-zowe/tree/fix/circular-imports 👍
I believe he will create an issue to track this in the repo.

**Comment by zFernand0** [comment]
merging! 🥳

**Pull request merged**